packages/notification-center/src/components/popover-notification-center/PopoverNotificationCenter.tsx

```diff
@@ -22,6 +23,10 @@ interface IPopoverNotificationCenterProps {
   tabs?: ITab[];
   showUserPreferences?: boolean;
   onTabClick?: (tab: ITab) => void;
+  offset?: number;
+  position?:
+    | PopoverProps['position']
+    | `${NonNullable<PopoverProps['position']>}-${NonNullable<Exclude<PopoverProps['placement'], 'center'>>}`;
```

Comment:
updated types to be synonymous with the Mantine V5 props,
This will be removed once we upgrade